css/ast/src/selector.rs

```diff
@@ -21,13 +30,18 @@ pub struct CompoundSelector {
     pub span: Span,
     /// "&"
     pub nesting_selector: Option<NestingSelector>,
-    pub combinator: Option<SelectorCombinator>,
     pub type_selector: Option<TypeSelector>,
     pub subclass_selectors: Vec<SubclassSelector>,
 }
 
+#[ast_node("Combinator")]
+pub struct Combinator {
+    pub span: Span,
+    pub value: CombinatorValue,
+}
+
 #[derive(StringEnum, Clone, Copy, Eq, PartialEq, PartialOrd, Ord, Hash, EqIgnoreSpan)]
-pub enum SelectorCombinator {
+pub enum CombinatorValue {
```

Comment:
Can you add a type alias named `SelectorCombinator` with `#[deprecated = "Use CombinatorValue instead"]`?

Reply:
I can do it, but if we will do major bump we can avoid to keep legacy stuff in code, I think it is normal to avoid keeping old stuff until we reached v1

Reply:
All crates exposing AST as a part of their API will never hit `v1`, so I think it's better to have one.

Reply:
@kdy1 I'm just worried that this way we may have a lot of garbage/legacy stuff in the future and as long as we are at an early stage, we can avoid this because our crates do not use many developers